### PR TITLE
Azure SQL MI supports maximum 100 DB per instance.

### PR DESCRIPTION
**Pull request by Ramudaykumar**
Azure SQL Managed Instance supports maximum 100 user databases per instance. Enforce the threshold limit at the time of DB selection for migration.

[Work item](https://github.com/microsoft/azuredatastudio/issues/23352)

Experience details:
1. Customers selects than Azure MI supported limit (100 DBs), dialog will show an error message and select button will be disabled.
DB Selection threshold Error (See 4 as 100, locally changed 100 to 4 to produce the error)

2. As soon as user selected DB count goes under limit, it will show an info message and select button will be enabled.
DB Selection proceed message

### CODE REVIEW

**Comment by Ramudaykumar** [comment]
https://github.com/microsoft/azuredatastudio/pull/24465

File: extensions/sql-migration/src/constants/strings.ts [hunk 1]
```diff
@@ -114,6 +114,10 @@ export const PERF_DATA_COLLECTION_ERROR = (serverName: string, errors: string[])
 };
 export const SKU_RECOMMENDATION_ASSESSMENT_ERROR_BYPASS = localize('sql.migration.wizard.sku.assessment.error.bypass', 'Check this option to skip assessment and continue the migration.');
 export const SKU_RECOMMENDATION_ASSESSMENT_ERROR_DETAIL = localize('sql.migration.wizard.sku.assessment.error.detail', '[There are no assessment results to validate readiness of your database migration. By checking this box, you acknowledge you want to proceed migrating your database to the desired Azure SQL target.]',);
+export const AZURE_SQL_MI_DB_COUNT_THRESHOLD_EXCEEDS_ERROR = (selectedDbCount: number): string => {
+	return localize('sql.migration.select.azure.mi.db.count.threshold.exceeds.error', 'Error: Azure SQL Managed Instance supports maximum {0} user databases per instance. Select {0} or less database(s) to proceed further.', selectedDbCount);
+};
+export const AZURE_SQL_MI_DB_COUNT_UNDER_THRESHOLD = localize('sql.migration.select.azure.mi.db.count.under.threshold', 'To proceed press select button.');
```
Comment (smartura) [inline extensions/sql-migration/src/constants/strings.ts:120]: Would it make sense to capitalize and quote select?
'To proceed, press "Select" button.'

**Review by smartura — APPROVED** [review]
Approved with a small suggestion.

**Comment by brian-harris** [comment]
please include a snapshot of your changes

**Review by krritik — APPROVED** [review]

**Pull request merged**